mempoolspace/client.go

```diff
@@ -21,7 +42,42 @@ func recommendedFee() (*Client, error) {
 	var client Client
 	json.Unmarshal(responseBody, &client)
```

Comment:
The client shouldn't hold the response, we should create a structure for each json response (each API) and deserialize to it. For example here we should create "RecommendedFeesResponse" struct and return it as the method result. 
the Client should only hold parameters that are common to all API requests such as baseURL for example.